### DISCUSSION TITLE
How to unlock the line where the cursor live in the center vertically？

### DISCUSSION THREAD

**Discussion by Rick-24** [JetBrains/ideavim#681] [Q&A]
![image](https://github.com/JetBrains/ideavim/assets/62324822/4a219402-22fb-491d-aff3-4eb8aa02d334)
![image](https://github.com/JetBrains/ideavim/assets/62324822/68a34995-3399-42e7-b172-19d127cc7ac9)
This is so inconvenient for document reading!
So how can I solve it?

**Comment by AlexPl292** [comment]
Hi, the shortcut for toggling rendered view can be found in the tooltip

<img width="302" alt="image" src="https://github.com/JetBrains/ideavim/assets/4203721/75da88fc-a70f-4921-99be-59682ed4a4da">
